anaconda.py

```diff
@@ -149,6 +149,11 @@ def setup_environment():
     else:
         os.environ["DISPLAY"] = ":%s" % constants.X_DISPLAY_NUMBER
 
+    # We run directly under systemd and tmux, so bash won't load the file for us, and libreport
```

Comment:
IIRC in some scenarios (s390) we don't run via TMUX and also sometimes without systemd (container & Live image generation - but this might be non-interactive so out of scope). That still should not break this code though. :)

Reply:
Ah, thank you! Yes, it should work, I think. I will change the comment, so it makes sense.